spring-boot-project/spring-boot/src/test/java/org/springframework/boot/context/logging/LoggingApplicationListenerTests.java

```diff
@@ -500,12 +501,23 @@ public void systemPropertiesAreSetForLoggingConfiguration() {
 	@Test
 	public void environmentPropertiesIgnoreUnresolvablePlaceholders() {
```

Comment:
This test case was passing before due to the issue in I reported in `SpringIterableConfigurationPropertySource`